rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -355,8 +345,29 @@ def add_batch(self, *args, **kwargs):
 
     @Deprecated(
         old="RepayBuffer.replay(num_items)",
-        new="RepayBuffer.sample(" "num_items)",
+        new="RepayBuffer.sample(num_items)",
         error=False,
     )
     def replay(self, num_items):
         return self.sample(num_items)
+
+    @ExperimentalAPI
```

Comment:
Let's remove the `@Experimental` decorator here (failed experiment: it's already deprecated :D ).